Fabulous.XamarinForms/samples/AllControls/AllControls/Samples/UseCases/AppTheming.fs

```diff
@@ -0,0 +1,46 @@
+namespace AllControls.Samples.UseCases
+
+open Xamarin.Forms
+open Fabulous.XamarinForms
+open AllControls.Helpers
+
+module AppTheming =
+    let pageBackgroundColor = AppThemeColor(Default = Color.White, Dark = Color.DarkGray)
+    let pageForegroundColor = AppThemeColor(Default = Color.Black, Dark = Color.White)
+    
+    type Msg =
+        | SetRequestedAppTheme of OSAppTheme
+        
+    type Model =
+        { CurrentAppTheme: OSAppTheme }
+    
+    let init () =
+        { CurrentAppTheme = Application.Current.RequestedTheme }
+    
+    let update msg model =
+        match msg with
+        | SetRequestedAppTheme newTheme ->
+            { model with CurrentAppTheme = newTheme }
+        
+    let appThemeView model =        
+        View.NonScrollingContentPage(
+            title = "AppTheme sample",
+            backgroundColor = pageBackgroundColor.Value,
+            content =
+                View.Label(
+                    text = sprintf "Current App Theme is %A" model.CurrentAppTheme
+                )
+        ) 
+    
+    let view model _ =
+        match Device.RuntimePlatform with
+        | Device.iOS | Device.Android ->
+            appThemeView model
+
+        | _ -> 
+            View.ContentPage(
+                View.StackLayout [
+                    View.Label(text = "Your Platform does not support RadioButton")
```

Comment:
```suggestion
                    View.Label(text = "Your Platform does not support AppTheming")
```
This looks like a simple copy-paste error from ```RadioButton.fs```.

Reply:
Thanks!